src/main/java/com/github/codeboyzhou/mcp/declarative/server/McpSyncServerToolRegister.java

```diff
@@ -11,11 +11,7 @@
 
 import java.lang.reflect.Method;
 import java.lang.reflect.Parameter;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Wildcard imports are not recommended because they might introduce potential issues such as naming conflicts or reduced code clarity.